docs/backup-restore.md

```diff
@@ -0,0 +1,109 @@
+# Backup and Restore TiDB Cluster
+
+## Overview
+
+TiDB Operator supports two kinds of backup:
+
+* [Full backup](#full-backup)(scheduled or ad-hoc) via [`mydumper`](https://www.pingcap.com/docs/dev/reference/tools/mydumper/), which helps you logical backup the TiDB cluster.
+* [Incremental backup](#incremental-backup) via [`TiDB-Binlog`](https://www.pingcap.com/docs/dev/reference/tools/tidb-binlog/overview/), which helps you synchronize the data in the TiDB cluster to other database or backup the data at real-time.
+
+Currently, tidb-operator only supports automatic [restore operation](#restore) for full backup taken by `mydumper`. Restore the backup data captured by `TiDB-Binlog` requires human intervention.
+
+## Full backup
+
+Full backup using `mydumper` to take the logical backup of TiDB cluster. The backup job will create a PVC to store backup data.
```

Comment:
```suggestion
Full backup uses `mydumper` to take the logical backup of a TiDB cluster. The backup job creates a PVC to store backup data.
```

Reply:
- How to understand the `to take` here? It's a little bit confusing.
- It would be better to add the full name to `PVC` by using parentheses, like `PVC (x x x)`.

Reply:
"Full backup uses `mydumper` to make a logical backup of a TiDB cluster"

Is this clearer?